simulators/server.py

```diff
@@ -21,15 +21,16 @@
 
 class BaseHandler(BaseRequestHandler):
     """This is the base handler class from which `ListenHandler` and
-    `SendHandler` classes are inherited. It only defines the custom
-    header and tail for accepting some commands not related to the system
-    protocol, and the `_execute_custom_command` method to parse the received
-    custom command.
-    I.e. a $system_stop! command will stop the server, a $error! command
-    will configure the system in order to respond with errors, etc.
-    """
+    `SendHandler` classes are inherited. It only defines the custom header
```

Comment:
First sentence is plural, second is singular

Reply:
The second sentence starts with `It`, that refers to the `BaseHandler` class only, not the other two mentioned classes.